float64 speed_over_water
float64 current_speed_lon
float64 current_speed_lat

float64[3] covariance
